core/templates/services/user.service.spec.ts

```diff
@@ -257,9 +257,8 @@ describe('User Service', function() {
       .respond(500, errorMessage);
 
     UserService.setProfileImageDataUrlAsync(newProfileImageDataurl)
-      /* eslint-disable dot-notation */
+      /* eslint-disable-next-line dot-notation */
```

Comment:
Why is this check needed at all?

(In general try to remove checks that aren't needed.)